Source/GameView/Repositories/GameAPIRepo.swift

```diff
@@ -0,0 +1,205 @@
+//
+//  GameAPIRepo.swift
+//  CommonFiles
+//
+//  Created by Tiewhan Smith on 2020/03/06.
+//
+
+import Foundation
+
+///Structure is only used to model JSON respones. Contains the a structure of Apps
+struct AppList: Codable {
+  let applist: Apps
+}
+
+///Structure is used to model JSON respones. Contains the list of games
+struct Apps: Codable {
```

Comment:
This struct seems a bit problematic as we have only one value in both this and AppList which contains the same name as the struct and does not read very well. Why don't we instead just have an AppList struct that contains a variable of type [JsonGame] as AppList does't seem to serve much purpose right now

Reply:
Ah, this is because JSON Decoder demands this. Applist mimics the JSON response format that is returned. hence the odd naming conventions such as appid. The member names have to match otherwise JSONDecoder fails completely

Reply:
Ahh okay, I've dealt with bad APIs before, you can use DTOs to get more ideal objects but this is fine then